Creation Sandbox/Assets/Scripts/VoiceSpawner.cs

```diff
@@ -63,55 +63,78 @@ private void OnSpeechInput(Data data)
         }
     }
 
-    void OnMessage(MessageResponse resp, string customData)
+    void OnMessage(object resp, string customData)
```

Comment:
Something you might want to try in your OnMessage to cast the generic object as a MessageResponse.

```
private void OnMessage(object resp, string data)
    {
        //  Convert resp to fsdata
        fsData fsdata = null;
        fsResult r = _serializer.TrySerialize(resp.GetType(), resp, out fsdata);
        if (!r.Succeeded)
            throw new WatsonException(r.FormattedMessages);

        //  Convert fsdata to MessageResponse
        MessageResponse messageResponse = new MessageResponse();
        object obj = messageResponse;
        r = _serializer.TryDeserialize(fsdata, obj.GetType(), ref obj);
        if (!r.Succeeded)
            throw new WatsonException(r.FormattedMessages);

        Log.Debug("ExampleConversation", "Conversation: Message Response | intent: {0} | confidence : {1}", messageResponse.intents[0].intent, messageResponse.intents[0].confidence);
    }
```